### DISCUSSION TITLE
"go fullscreen" is now "distraction-free mode"

### DISCUSSION THREAD

**Discussion by danielweck** [edrlab/thorium-reader#2355] [Ideas]
<img width="968" alt="Screenshot 2024-05-22 at 13 54 26" src="https://github.com/edrlab/thorium-reader/assets/625201/ccb1e285-b8ac-4a98-8f4c-424d9b32d06f">

**Comment by danielweck** [comment]
"zen" mode currently offers zero graphical affordances other than the fullscreen icon button itself, and the "skip to content" blue link in the upper corner (reachable via keyboard tab)

Shouldn't we expose at least the previous/next buttons? In Thorium2 there was a timer to automatically hide GUI when the mouse cursor is immobile (we still do this for the mouse cursor itself)

<img width="1001" alt="Screenshot 2024-05-22 at 13 59 06" src="https://github.com/edrlab/thorium-reader/assets/625201/0a2c3437-da20-44c1-bb7c-d40bffb74c2b">

**Comment by llemeurfr** [comment]
I personally like the distraction free result. Most users, on a computer, user right-left arrows without thinking. 
I propose to wait for feedbacks on v3.0 and advise.

**Comment by llemeurfr** [comment]
Therefore I think this issue should be moved to a discussion.

**Comment by danielweck** [comment]
I agree about moving this issue to a discussion about the UX design of the "distraction-free" (previously "fullscreen") feature, which in my opinion is currently incomplete / ill-designed.

The current implementation is not usable by mouse-centric (pointing device) users, as the previous/next commands can only be invoked via keyboard shortcuts / key strokes. Touch screen support is not guaranteed so we cannot recommend finger swipe interactions as an alternative.

My recommendation is to display the arrow icons on the left/right sides of the content viewport, just as we do in the normal mode. The mouse cursor already auto-hides after a few seconds and reappears on mouse move, I think a similar UX pattern can be implemented for the icon arrow buttons on either side of the document content.
